src/app/DTT/text/index.js

```diff
@@ -19,7 +19,10 @@ export const Write = ({ questionCode, data, onSendAnswer, regexPattern, errorMes
   const [userInput, setuserInput] = useState(data?.value)
 
   try {
+    // console.log('PATTERN:', regexPattern)
```

Comment:
Please remove the console logs if they are not being used rather than just commenting them out.